xtext-website/documentation/310_eclipse_support.md

```diff
@@ -781,6 +781,22 @@ The base class `AbstractXtextCodeMiningProvider` provides some factory methods f
 
 For an implementation reference, have a look at the Xtext Domainmodel example.
 
+## UI Testing {#ui-testing}
+
+Automated UI tests are crucial for the maintanability and the quality of a software product. That is why it is strongly recommended to write not only automated unit tests for your language, but also automated UI tests for your language editor. The `org.eclipse.xtext.ui.testing` package provides some abstract base classes: 
```

Comment:
There is a typo in 'maintanability' -> maintainability

Reply:
That is -> That's ?

Reply:
Maybe remove "abstract" here, it should be clear from their names

Reply:
Fixed.